mappings/empire_v2.0.yaml

```diff
@@ -0,0 +1,341 @@
+model:
+  - EMPIRE v2.0
+
+native_regions:
+  - Austria
+  - Bosnia and Herzegovina
+  - Belgium
+  - Bulgaria
+  - Switzerland
+  - Czech Republic
+  - Germany
+  - Denmark
+  - Estonia
+  - Spain
+  - Finland
+  - France
+  - United Kingdom
+  - Greece
+  - Croatia
+  - Hungary
+  - Ireland
+  - Italy
+  - Lithuania
+  - Luxembourg
+  - Latvia
+  - North Macedonia
+  - The Netherlands
+  - Norway
+  - Poland
+  - Portugal
+  - Romania
+  - Serbia
+  - Sweden
+  - Slovenia
+  - Slovakia
+  - Norway|Ostland: EMPIRE v2|Norway|Ostland
+  - Norway|Sorland: EMPIRE v2|Norway|Sorland
+  - Norway|Norgemidt: EMPIRE v2|Norway|Norgemidt
+  - Norway|Troms: EMPIRE v2|Norway|Troms
+  - Norway|Vestmidt: EMPIRE v2|Norway|Vestmidt
+  - North Sea|Moray Firth: EMPIRE v2|North Sea|Moray Firth  
+  - North Sea|Firth of Forth: EMPIRE v2|North Sea|Firth of Forth  
+  - North Sea|Dogger Bank: EMPIRE v2|North Sea|Dogger Bank  
+  - North Sea|Hornsea: EMPIRE v2|North Sea|Hornsea  
+  - North Sea|Outer Dowsing: EMPIRE v2|North Sea|Outer Dowsing  
+  - North Sea|Norfolk: EMPIRE v2|North Sea|Norfolk  
+  - North Sea|East Anglia: EMPIRE v2|North Sea|East Anglia  
+  - North Sea|Borssele: EMPIRE v2|North Sea|Borssele  
+  - North Sea|Hollandsee Kust: EMPIRE v2|North Sea|Hollandsee Kust  
+  - North Sea|Helgolander Bucht: EMPIRE v2|North Sea|Helgolander Bucht  
+  - North Sea|Nordsoen: EMPIRE v2|North Sea|Nordsoen  
+  - North Sea|Utsira Nord: EMPIRE v2|North Sea|Utsira Nord  
+  - North Sea|Sorlige Nordsjo I: EMPIRE v2|North Sea|Sorlige Nordsjo I  
+  - North Sea|Sorlige Nordsjo II: EMPIRE v2|North Sea|Sorlige Nordsjo II  
+
+
+# allow directional connections explicitly
+- Austria>Czech Republic  
+- Austria>Germany  
+- Austria>Hungary  
+- Austria>Italy  
+- Austria>Slovakia  
+- Austria>Slovenia  
+- Austria>Switzerland  
+- Belgium>Borssele  
+- Belgium>France  
+- Belgium>Germany  
+- Belgium>Great Britain  
+- Belgium>Hollandsee Kust  
```

Comment:
In the result data, the nodes are treated as separate countries. I have to double check with Mostafa if this then makes sense then since in your approach A that you mentioned in the email (which Mostafa told me to implement) they are all part of the North Sea.

Reply:
I just asked Mostafa and that change is fine. I will change it in the results.